Repository/DbModels/DbModels.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Model;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Repository.DbModels
+{
+    public class DbModels : DbContext
+    {
+        public DbModels(DbContextOptions<DbModels> options) : base(options)
+        {
+        }
+
+        public DbSet<Course> Courses { get; set; }
+        public DbSet<Enrollment> Enrollments { get; set; }
+        public DbSet<User> Users { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Course>().ToTable("Course");
```

Comment:
This is done if you want your table name to be different from class name, but you named table just like your class and that is default behaviour, so you can delete this